testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -71,6 +73,16 @@
 @ExtendWith(CqlSessionExtension.class)
 @CqlSessionSpec()
 public class RestApiv2Test extends BaseIntegrationTest {
+  // Due to Guava's null hatred, ImmutableMap cannot contain nulls.
+  // But we have a need for null values in JSON so here's something that
+  // will get serialized as JSON null.
+  private final Object GUAVA_NULL_VALUE =
+      new Object() {
+        @JsonValue
```

Comment:
Means literally "use return value of this method as value to actually serialize": Jackson calls method, gets `null`, serializes as JSON null.

Reply:
Why not using a simple `NullNode.getInstance()`? You anyway have `Map<String, Object>` in the rows creation?

Reply:
Actually, good point, will change.

Yes, plain `Map` used in many places over ImmutableMap. So perhaps better avoid its use here in general.